webpack.config.js

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+import webpack from 'webpack';
+import UglifyJSPlugin from 'uglifyjs-webpack-plugin';
+
+import {settings} from './gulp/config';
+
+const dev = {
+	entry: {
+		polyfills: './src/app/polyfills.js',
+		app: './src/app/index.js',
+		vendor: settings.sources.externalJs
+	},
+	output: {
+		path: path.join(__dirname, settings.destinations.dev.app),
+		filename: '[name].bundle.js'
+	},
+	plugins: [
+		new webpack.optimize.CommonsChunkPlugin({
+			name: ['app', 'vendor', 'polyfills']
+		}),
+		new webpack.ProvidePlugin({
+			$: 'jquery',
+			jQuery: 'jquery'
+		})
```

Comment:
Like @phlmn already said this is something one should be able to config via the `baumeister` section within the `package.json`.

Proposal:

```
"baumeister": {
  "bundleCSS": [],
  "bundleExternalJS": [
    "jquery",
    "bootstrap-sass"
  ],
  "autoImport": {
    "$": 'jquery',
    "jQuery": 'jquery'
  },
  "includeStaticFiles": [
    "bootstrap-sass/assets/fonts/**/*"
  ]
}
```

I guess `autoImport` or `autoLoad` is more self-explaining than `provide`. Although it needs to be added to the README and should link to https://webpack.js.org/plugins/provide-plugin/.

Any other ideas?

Reply:
Maybe its better to name thinks to what they are 🤔 

Makes it easier for users who already have worked with webpack.

```json
"baumeister": {
  "webpack": {
    "DefinePlugin": {
      "dev": {},
      "prod": {
        "process.env": {
          "NODE_ENV": "'production'"
        }
      }
    },
    "ProvidePlugin": {
      "$": "jquery",
      "jQuery": "jquery"
    }
  }
}
```